gitignore

```diff
@@ -0,0 +1,225 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
This file `gitignore` should be removed.

Reply:
don't see any diff from the master

Reply:
Because this file never existed before.